pkg/providers/cloudstack/validator.go

```diff
@@ -139,15 +138,6 @@ func (v *Validator) ValidateClusterMachineConfigs(ctx context.Context, clusterSp
 	}
 
 	for _, machineConfig := range clusterSpec.CloudStackMachineConfigs {
-		if len(machineConfig.Spec.Users) <= 0 {
-			machineConfig.Spec.Users = []anywherev1.UserConfiguration{{}}
-		}
-		if len(machineConfig.Spec.Users[0].SshAuthorizedKeys) <= 0 {
-			machineConfig.Spec.Users[0].SshAuthorizedKeys = []string{""}
-		}
-		if machineConfig.Spec.Users[0].Name == "" {
-			machineConfig.Spec.Users[0].Name = v1alpha1.DefaultCloudStackUser
-		}
```

Comment:
These aren't validations, so moving this logic to the SetupAndValidate provider methods